control-plane-operator/controllers/hostedcontrolplane/snapshotcontroller/testdata/zz_fixture_TestReconcileOperatorDeployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apps/v1
```

Comment:
I'm missing where this is being consumed?

Reply:
In an unit tests here: https://github.com/openshift/hypershift/pull/1698/files#diff-a2a76e7d0407603302ee6d2eb3852dbcceab3a475b120580b6cd6fc214174c7eR43

```
testutil.CompareWithFixture(t, deploymentYaml)
```